modules/ossm-cr-example.adoc

```diff
@@ -182,98 +182,98 @@ The following table lists the specifications for the `ServiceMeshControlPlane` r
 |===
 |Name |Description |Configurable parameters
 
-|addons
-|Addon is used to configure additional features beyond core control plane components, such as visualization, or metric storage.
+|`addons`
+|`Addons` is used to configure additional features beyond core control plane components, such as visualization, or metric storage.
```

Comment:
If you're going to format all of the parameter instances, then we need to make them lower case (to avoid confusion around the proper case) and rewrite the sentences so we're not starting with a lower case word.

Reply:
Ok. I can do that. What should be the behavior in a case like this? Is it a grey area? I.e. MyParameter | MyParameter is used for foo. Should MyParameter be formatted in both places? I've been understanding it as if you're referring to the parameter, it should be formatted, if not then it doesn't need to be.

Reply:
We should probably format the parameters in both places and revise the description to be something like "You use the `addons` parameter to configure additional features beyond core control plane components, such as visualization, or metric storage."  (which also solves the current passive voice problem by making it active voice).

Reply:
Updated. Thanks. Please take a look and let me know what you think.

Reply:
LGTM

Reply:
Thanks